CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.8.0
+
+* zql: add the `put` proc that adds or updates fields using a computed
+  expression. (#437)
+* zql: add functions for use with put, like `Math.min`, `Math.max`, and others.
+  (#453, #459, #461, #472)
+* zq: support reading ndjson with user supplied type information. (#441)
+* Fix an issue reading pcaps generated via NetScout. (#462)
```

Comment:
```suggestion
* Fix an issue with reading pcaps with snaplen=0. (#462)
```
While the initially reported issue was regarding NetScout, I'm in favor of keeping this generic.